__std__/nix/toolchain/devshellProfiles/common.nix

```diff
@@ -0,0 +1,16 @@
+{ inputs, cell }:
+
+_: {
+  commands = [
+    {
+      package = cell.packages.git-work-in-progress;
```

Comment:
do we want to commit this? or is this something that Lorenzo likes? ;)

Reply:
I actually stole the idea from one of you guys after seeing a massive pile of "WIP" commits in a PR. 
Can't remember who and where.

Anyway, to answer your questions: Yes and Yes? 😁